plugin/core/edit.py

```diff
@@ -54,9 +55,12 @@ def apply_text_edits(
     label: str | None = None,
     process_placeholders: bool | None = False,
     required_view_version: int | None = None
-) -> None:
+) -> Promise[sublime.View]:
     if not edits:
-        return
+        return Promise.resolve(view)
+    if not view.is_valid():
+        print('LSP: ignoring edits due to view not being open')
+        return Promise.resolve(view)
```

Comment:
I suppose it's a bit weird to ignore edits but still return a view for something down the chain. But returning `None` could make usage annoying on the other hand.